docs/getting-started.md

```
@@ -0,0 +1,162 @@
+**Quick Navigation**:
+[[toc]]
+
+## Getting the templates
+
+There are two different ways on how to get started with using the templates. Which variant to use depends on whether you
+want to create a completely new project or add new components to an existing code repository.
+
+### Variant 1: Use GitHub template mechanism
+
+When starting a completely new project on GitHub, it's recommended to use the "template repository" feature of GitHub.
+The `refarch-templates` repository under https://github.com/it-at-m/refarch-templates is defined as a "template repository".
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)